source/_integrations/climate.modbus.markdown

```diff
@@ -32,6 +32,7 @@ climate:
     temp_step: 1
     target_temp_register: 2782
     current_temp_register: 27586
+    current_temp_register_type: holding
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
OK, I will update that. Thanks!